biz.aQute.repository/src/aQute/maven/provider/POM.java

```diff
@@ -72,7 +75,12 @@ public POM() {
 	}
 
 	public POM(MavenRepository repo, InputStream in) throws Exception {
-		this(repo, getDocBuilder().parse(processEntities(in)));
+		this(repo, getDocBuilder().parse(processEntities(in)), false);
+	}
+
+	public POM(MavenRepository mavenRepository, InputStream pomFile, boolean b)
```

Comment:
Can the arg name `b` be made more descriptive? I think it should be `ignoreIfParentAbsent`.